Runtime/AvatarCreator/Scripts/UI/Elements/DeleteAvatarElement.cs

```diff
@@ -21,12 +22,20 @@ public class DeleteAvatarElement : MonoBehaviour
 
         private string avatarId;
         private AvatarAPIRequests avatarAPIRequests;
+        private CancellationTokenSource cancellationTokenSource;
```

Comment:
why not just do this? 

private CancellationTokenSource cancellationTokenSource = new CancellationTokenSource();

Reply:
good point 👍